pandas/_config/config.py

```diff
@@ -763,10 +763,12 @@ def inner(key: str, *args, **kwds):
     set_option = wrap(set_option)
     get_option = wrap(get_option)
     register_option = wrap(register_option)
-    yield None
-    set_option = _set_option
-    get_option = _get_option
-    register_option = _register_option
+    try:
+        yield None
```

Comment:
nitpick can this just be "yield" like the others

Reply:
done